pages/1.13/release-notes/1.13.1/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.1
+title: Release notes for 1.13.1
+menuWeight: 1
+excerpt: Release notes for DC/OS 1.13.1, including Open Source attribution, and version policy.
+---
+DC/OS 1.13.1 was released on May 31, 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.13.1/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="http://downloads.mesosphere.com/dcos-enterprise/stable/1.13.1/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the <a href="https://support.mesosphere.com/s/downloads">support website</a>. For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+This release provides enhancements and fixes to address reported issues, integrate changes from previous releases, and maintain compatibility and support for the DC/OS ecosystem.
+
+If you have DC/OS deployed in a production environment, see [Known issues and limitations](#known-issues) to see if any potential operational changes for specific scenarios apply to your environment.
+
+# Issues fixed in this release
+The issues that have been fixed in DC/OS 1.13.1 are grouped by feature, functional area, or component. Most change descriptions include one or more issue tracking identifiers enclosed in parentheses for reference.
+
+### Admin Router
+- Changes the maximum size allowed for uploads to a service through Admin Router (COPS-4651, DCOS-52768).
+
+    This release increases the maximum size allowed for uploading packages from 1GB to 16GB. This change enables you to upload larger packages to a registry service without timing out the upload connection.
+
+<!--### Command-line interface (CLI)
+- 
+
+### Diagnostics and logging
+- 
+
+### GUI --> 
```

Comment:
We're missing a RN for the fix for COPS-4920 " UI does always show task status for tasks that were running before an upgrade".  Merge was done here: https://github.com/dcos/dcos/pull/5585/files

Customers have encountered this issue so I think it's important to mention it in the RN.

Reply:
Added.